packages/mobile/src/send/Send.tsx

```diff
@@ -112,8 +111,6 @@ function Send({ route }: Props) {
     (recipient: Recipient) => {
       dispatch(hideAlert())
 
-      dispatch(storeLatestInRecents(recipient))
-
```

Comment:
Sure we want to remove this?
I guess it depends if we want recents to appear only if the user actually sent.

Reply:
yeah, I've found it really annoying when I enter an address or select a contact, but I don't actually send them anything, and then they appear in my Recent list

Reply:
ok, if you haven't already please check this change of behavior with @coreycelo or @aleksey-nov too.